kfs-core/src/main/resources/edu/arizona/kfs/workflow/PaymentRequestDocument.xml

```diff
@@ -14,6 +14,7 @@
 	      	<status>Initiated</status>
 	      	<status>In Process</status>
 	      	<status>Canceled In Process</status>
+	      	<status>Canceled - In Process</status>
```

Comment:
Based on the values in DB table DEPR_AP_PMT_RQST_STAT_T and the values in org/kuali/kfs/module/purap/PurapConstants.java, there is only one "canceled in process" valid PURAP document status code for the Payment Request Document. Therefore, "Canceled - In Process" (line 17) should replace "Canceled In Process" (line16).